src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/App/index.js

```diff
@@ -24,10 +25,13 @@ export const App = ({ isCernerPatient }) => {
 App.propTypes = {
   // From mapStateToProps.
   isCernerPatient: PropTypes.bool,
+  showNewGetMedicalRecordsPage: PropTypes.bool.isRequired,
```

Comment:
I _think_ this will be null when the feature flags are loading from the API. Personally I wouldn't bother with the strict `=== false` check. I'd just treat the loading state the same as the feature-off state

Reply:
I was thinking I'd merge it in before today's deploy originally, so now that we're planning on merging this after today's deploy that _could_ work but that means that by tomorrow's deploy all pages will have the new unauth content in production, whereas the release is supposed to happen Friday 🤔